### PR TITLE
Iati data summary

### PR DESCRIPTION
**Pull request by BibianaC**
Add more information in Data Summary:
- the name of the file / input source you provided
- the dateTime you did this
- the number of records we found (which is a count of the unique iati-identifier and organisation_identifier instances)

Screen Shot 2019-03-27 at 15 54 21
Screen Shot 2019-03-27 at 15 55 11

### CODE REVIEW

File: requirements.in [hunk 1]
```diff
@@ -1,9 +1,9 @@
-pip
+pip==9.0.3
```
Comment (Bjwebb) [inline requirements.in]: What's the purpose of this?

Comment (BibianaC) [reply requirements.in]: I could put it back as it was, in fact I was not intent to push this change. The lastest pip version 19.0.3 has a bug and every time I update requirements and it upgrades to the latest version I have many problems. I need to rollback.

Comment (Bjwebb) [reply requirements.in]: I think it's okay to leave this in, but it probably wants an explanation in a comment. Is there an upstream issue to link to?

Comment (BibianaC) [reply requirements.in]: I have not seen any issue about this.

File: cove_iati/lib/iati.py [hunk 2]
```diff
@@ -421,3 +426,21 @@ def get_file_type(file):
...
+    tree = get_tree(data_file)
+    root = tree.getroot()
+    identifiers = root.xpath('/iati-activities/iati-activity/iati-identifier/text()')
+    unique_identifiers = list(set(identifiers))
+
+    return len(unique_identifiers)
+
+
+def organisation_identifier_count(data_file):
+    tree = get_tree(data_file)
```
Comment (Bjwebb) [inline cove_iati/lib/iati.py]: I think it would be better to avoid opening the file multiple times, by passing around the tree object.

Comment (BibianaC) [reply cove_iati/lib/iati.py]: Yes, you are right. I have done an amendment to that.

**Pull request merged**